src/geometries/polytopes/pyramid.jl

```diff
@@ -11,6 +11,11 @@ A pyramid with points `p1`, `p2`, `p3`, `p4`, `p5`.
 
 nvertices(::Type{<:Pyramid}) = 5
 
+function base(p::Pyramid)
+  a, b, c, d, o = vertices(p)
```

Comment:
```suggestion
  a, b, c, d, o = p.vertices
```

Given that this is a function/accessor implemented in the same file of the geometry, we tend to use internal field access. There is no clear rule, but I am trying to follow the pattern in other polytope implementations.

Reply:
I've updated the file. This line, and another function as well.